pkgs/build-support/fetchrepoproject/test.nix

```diff
@@ -0,0 +1,12 @@
+with import <nixpkgs> {};
+let fetcher = import ./default.nix {inherit stdenv cacert
+git gitRepo gnupg;};
+in
+  fetcher {
+  name = "foo";
+  manifest = "https://github.com/CopperheadOS/platform_manifest.git";  
+  rev="nougat-mr1.1-release";
+  sha256 = "51rhjsqka5v02zw6k0m4fnwz8xrx70ifar4afsndm88i6fbd3vaw"; #fake hash, don't worry
```

Comment:
the hash is just a random hash to test that repo actually does its thing (that manifest branch, without a tag, follows HEAD)... also it fetches about 80GB of sources |-)

I could change it to a smaller project with a fixed rev, though?